.evergreen/scripts/install-uninstall-check.sh

```diff
@@ -10,19 +10,19 @@ DIR=$(dirname $0)
 . $DIR/find-cmake-latest.sh
 CMAKE=$(find_cmake_latest)
 . $DIR/check-symlink.sh
+SRCROOT=$(pwd)
 
-if command -v gtar 2>/dev/null; then
-  TAR=gtar
-else
-  TAR=tar
-fi
+SCRATCH_DIR=$(pwd)/.scratch
+rm -rf "$SCRATCH_DIR"
+mkdir -p "$SCRATCH_DIR"
+cp -vr -- "$SRCROOT"/* "$SCRATCH_DIR"
```

Comment:
Consider using `cmake -S . -B "$BUILD_DIR"` and `cmake --build "$BUILD_DIR"` instead of copying the source root directory and running CMake commands with relative paths from inside `$BUILD_DIR`.